ROADMAP.adoc

```diff
@@ -0,0 +1,40 @@
+= jenkins.io roadmap
+
+This document is the home of the roadmap for moving from the "legacy" site on link:https://jenkins-ci.org[jenkins-ci.org]
+to the new and improved, super amazing, link:https://jenkins.io[jenkins.io]
+
+
+== Static Site switchover
+
+1. properly import non-story (see standalone) pages from Drupal (link:https://issues.jenkins-ci.org/browse/WEBSITE-34[WEBSITE-34])
+1. import legacy site (*on-going until full cut-off*)
+1. determine where we can generate the site (link:https://issues.jenkins-ci.org/browse/WEBSITE-35[WEBSITE-35])
+1. determine how we're going to host (link:https://issues.jenkins-ci.org/browse/WEBSITE-38[WEBSITE-38])
+1. deploy the new statically generated site to link:http://beta.jenkins-ci.org[beta.jenkins-ci.org]
+1. cut over from Drupal to the statically generated site
+
+== Development of new site
+
+Developing the new site should, ideally occur in the same branch as the static legacy site. This will help ensure that,
```

Comment:
FWIW, I'm of the same opinion as @daniel-beck, but Tyler already knows that, and I don't want to block him. My impression is that @rtyler has some ideas about how to build the same source twice in different ways to produce both the "1.0" site with old L&F and a subset of content, and the "2.0" site with new L&F.

So my hopefully constructive suggestion here is that if that's an intent, I'd help me to see that called out. Maybe a section about "Working on jenkins.io while keeping jenkins-ci.org"?